discover/analytics-cloud/articles/01-what-is-liferay-analytics-cloud.markdown

```diff
@@ -0,0 +1,124 @@
+# What is Liferay Analytics Cloud?
+
+Your company interacts with lots of people, including existing and prospective
+customers. On your websites they browse products and services, ask questions,
+leave feedback, etc. Presenting relevant, interesting website content motivates
+people to do business with you. Knowing your customers well and understanding
+how they interact with existing digital content can help you improve their
+experience. Liferay Analytics Cloud generates powerful, intuitive charts that
+deepen your knowledge of people and their online behavior so you can create
+terrific user experiences that drive business. 
+
+It all starts with understanding people.
+
+## Understanding People 
+
+Existing and prospective customers interact with your company in person, on your
+websites, and via email and messaging. On Liferay Digital Enterprise (DE)
+websites, people are registered users or visitors. Liferay Analytics Cloud
+(Analytics Cloud) integrates contacts from your Liferay DE instances and CSV
+files into a single view. Analytics Cloud merges contact data into rich profiles
+of *Individuals*---unique contacts (people). Analytics Cloud lets you name
+contact fields your way, making them most useful to you and your company.
+
+Analytics Cloud lets you aggregate Individuals into *Segments*. Segments are
```

Comment:
Too many sentences in a row starting with "Analytics Cloud." Reword these for better flow.